src/Controller/ReleasesController.php

```diff
@@ -19,6 +19,9 @@ class ReleasesController extends AppController
     public function index()
     {
         $releases = $this->Releases->find('all')
+            ->where([
+                'Releases.release_date <=' => date('Y-m-d')
```

Comment:
Will that be enough, shouldn't we add an hour as well to make it accessible at the end of the BoF?